### PR TITLE
LPS-78974 New Lexicon localization pattern support for text fields and image description

### PR DESCRIPTION
**Pull request by pavel-savinov**

### CODE REVIEW

**Comment by pavel-savinov** [comment]
cc @JorgeFerrer

**Comment by pedroqueiroz** [comment]
ci:test

**Comment by pedroqueiroz** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js [hunk 1]
```diff
@@ -500,13 +500,27 @@ AUI.add(
...
-								instance.get('name'),
-								INSTANCE_ID_PREFIX,
-								instance.get('instanceId')
-							]
-						).join('');
+						if (instance.get('localizable') && component) {
+							return prefix.concat(
+								[
+									instance.get('name'),
+									'_',
```
Comment (pedroqueiroz) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js]: Hey @pavel-savinov , why are these extra underscores necessary?

Comment (pavel-savinov) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js]: @pedroqueiroz They come from input-localized taglib, see https://github.com/pedroqueiroz/liferay-portal/blob/master/portal-web/docroot/html/taglib/ui/input_localized/page.jsp#L298

**Comment by pedroqueiroz** [comment]
Pull request submitted to https://github.com/rafaprax/liferay-portal/pull/423. [See changes here.](https://github.com/pedroqueiroz/liferay-portal/compare/9377aa531328~1...9377aa531328) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**